app/javascript/gobierto_data/webapp/components/sets/VisualizationsList.vue

```diff
@@ -118,6 +108,7 @@ import Visualizations from "./../commons/Visualizations.vue";
 import PrivateIcon from './../commons/PrivateIcon.vue';
 import { Dropdown } from "lib/vue-components";
 import { getUserId } from "./../../../lib/helpers";
+import VisualizationsSlot from "./../commons/VisualizationsSlot.vue";
```

Comment:
Since you're trying to display some kind of card, I'd strongly suggest to use that name as the component one: **Card**
Also, this component is a layout, because it does do nothing but wrap the code with divs and classes, so move it to _Layouts_ folder